### ISSUE TITLE
Joi couldn't used directly

### ISSUE THREAD

**Issue by dspangenberg** [roboncode/orango#19]
```
let schema = orango.Schema(
 {
    firstName: Joi.string(),
    name: String,
    createdAt: { type: Date },
    updatedAt: { type: Date, default: Date.now }
  }
```
Uncaught TypeError: Cannot read property 'type' of null

**Comment by roboncode** [comment]
Verified. Thx. I will work on fixing this.

**Comment by roboncode** [comment]
Fixed and pushed to master from commit #26 Below is the test I performed:

```js
await orango.connect()

const UserSchema = orango.Schema({
  name: Joi.string()
})

let User = await orango.model('User', UserSchema)
let user = await new User().save()
console.log('User created'.green, user)
```
